Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/Controller.mo

```diff
@@ -0,0 +1,107 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Relay;
+block Controller
+  "Outputs relay signals for tuning PID controllers"
+  parameter Real yHig(min=1E-6) = 1
+    "Higher value for the output";
+  parameter Real yLow(min=1E-6) = 0.5
+    "Lower value for the output";
+  parameter Real deaBan(min=1E-6) = 0.5
+    "Deadband for holding the output value";
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput u_s
+    "Connector for the setpoint input signal"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+    iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput u_m
+    "Connector for the measurement input signal"
+    annotation (Placement(transformation(origin={0,-120},extent={{20,-20},{-20,20}},rotation=270),
+    iconTransformation(extent={{20,-20},{-20,20}},rotation=270,origin={0,-120})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput y
+    "Control output signal"
+    annotation (Placement(transformation(extent={{100,50},{120,70}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput On
+    "Control switch signal"
+    annotation (Placement(transformation(extent={{100,-90},{120,-70}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yErr
+    "Control error signal"
+    annotation (Placement(transformation(extent={{100,-10},{120,10}})));
+  Buildings.Controls.OBC.CDL.Logical.OnOffController greMeaSet(final bandwidth=deaBan*2, pre_y_start=true)
```

Comment:
when assigning the parameters, change line, add `final`

Reply:
ditto